src/cockpit/389-console/src/lib/server/settings.jsx

```diff
@@ -850,15 +858,17 @@ export class ServerSettings extends React.Component {
                                 min={4096}
                                 max={9223372036854775807}
                                 onMinus={() => { this.onMinusConfig("nsslapd-disk-monitoring-threshold", "diskmon") }}
-                                onChange={(e) => { this.onConfigChange(e, "nsslapd-disk-monitoring-threshold", 4096, 9223372036854775807, "diskmon") }}
+                                onChange={(e) => { this.onConfigChange(e, "nsslapd-disk-monitoring-threshold", 1, 9223372036854775807, "diskmon") }}
                                 onPlus={() => { this.onPlusConfig("nsslapd-disk-monitoring-threshold", "diskmon") }}
                                 inputName="input"
                                 inputAriaLabel="number input"
                                 minusBtnAriaLabel="minus"
                                 plusBtnAriaLabel="plus"
                                 widthChars={8}
-                                validated={this.state.errObjDiskMon.diskThreshold ? ValidatedOptions.error : ValidatedOptions.default}
                             />
+                            <FormHelperText isError isHidden={!this.state.errObjDiskMon['nsslapd-disk-monitoring-threshold']}>
+                                Value must be greater than or equal to 4096
```

Comment:
Same here.

Reply:
This works for me, no error with 4096 (as expected), but I did not try saving it...